react/components/AvailabilitySubscriber/mutations/addToAvailabilitySubscriberMutation.gql

```diff
@@ -0,0 +1,9 @@
+mutation availabilitySubscribe($acronym: String, $document: DocumentInput)
+  @context(scope: "private") {
```

Comment:
You don't need this. the context is now define on `store-graphql`, right @tlgimenes ?

Reply:
Yes, you don't need this anymore. All mutations are private now

Reply:
I just copy paste from BuyButton, maybe is outdate too.